robottelo/host_helpers/contenthost_mixins.py

```diff
@@ -89,6 +89,26 @@ def enable_rhel_repo(self, organization_id):
             releasever=None,
         )
 
+    def create_repo(self, rpm_name, repo_path, post_upgrade=False, other_rpm=None):
```

Comment:
Ahh! I was searching for similar function if we have it , but I couldnt find.

I will see if we can merge this with the existing one.

Reply:
So I determine that the `custom_repo` def here is a different than what you have linked.

This def actually creates a repo from a given rpm on `/var/www/html/pub/` direction to be exposed over HTTP so that it could be used in Satellite itself to create repo e.g `entities.Repository(url='http://satellite_url/pub/custom_repo').create()`
Also, its very customizable from upgrade tests point of view to update repomd post upgrade and remove existing rpm.

But the custom_repo def you point is actually used in Satellite system to create repository in `/etc/yum.repos.d/custom_repo.repo` so that could be used to install rpm in satellite server.

So my decision is I rename this repo to `create_custom_html_repo` so that's more verbose.

Reply:
Ping @JacobCallahan !